Config/Inc/config.h

```diff
@@ -20,7 +20,7 @@ typedef enum SafetyStatus_e {SAFE = 0, DANGER = 1, OVERVOLTAGE = 2, UNDERVOLTAGE
 
 //--------------------------------------------------------------------------------
 // Basic Parameters of BPS layout
-#define NUM_MINIONS	4					 // Number of minion boards
+#define NUM_MINIONS	1					 // Number of minion boards
```

Comment:
This is fine for testing, but shouldn't be committed.

Let's make sure that the config.h file stays proper for a production build.

Reply:
Good catch